modular_skyrat/modules/sec_haul/code/corrections_officer/Space_Station_13_areas.dm

```diff
@@ -1,4 +1,4 @@
-/area/corrections_officer
+/area/station/security/corrections_officer
 	name = "Corrections Officer Lounge"
 	icon = 'modular_skyrat/modules/skyrat_areas/icons/areas_station.dmi'
 	icon_state = "corrections_officer"
```

Comment:
Update paths since you're changed the type definition, and this'll create issues on BlueShift and Deltas prison slot in